packages/storybook/src/nimble/table/table.stories.ts

```diff
@@ -494,6 +511,7 @@ export const table: StoryObj<TableArgs> = {
         validity: undefined,
         checkValidity: undefined,
         tableRef: undefined,
+        fitRowsHeight: true,
```

Comment:
I'm on the fence of making this default true. I think it's generally a special case usage and more tempted to default to false to better represent the out of box behavior. Could be convinced either way.

Reply:
I can be convinced either way as well. I decided to default it to `true` because it made the docs page read better without reserving a bunch of extra space for the table when the default data set has only a few rows. But, I agree that the common case will be not to use the new token.

Reply:
I did find it harder to read the docs for the large data set case since you have to scroll past a full screen of rows at that point. So if we do go this way lets set a max-height that's smaller.

Reply:
Went ahead and made it default to `false` to keep existing behavior default.